geode-core/build.gradle

```diff
@@ -86,19 +86,19 @@ dependencies {
 
   compile 'org.apache.logging.log4j:log4j-api:' + project.'log4j.version'
   compile 'org.apache.logging.log4j:log4j-core:' + project.'log4j.version'
-  runtime ('org.fusesource.jansi:jansi:' + project.'jansi.version') {
```

Comment:
Why is this changed?  This should be on the class path.

Reply:
It is, but now as a runtimeOnly not runtime (deprecated).